docs/simpleinjector/metricsCapturing/installation.md

```diff
@@ -0,0 +1,17 @@
+# Installation
+
+Installation and decorator code for metrics-capturing is defined by the `Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing` NuGet package.  It functions as an extension of the `Functional.CQS.AOP.IoC.SimpleInjector` package.
+
+Rather than reference your application configuration file directly, `Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing` requires that a [`MetricsCapturingModuleConfigurationParameters`](../../../src/Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing/Configuration/MetricsCapturingModuleConfigurationParameters.cs) instance be supplied as part of the installation process.  Your application configuration can be any format you choose.  You can also supply the configuration values directly in code rather than loading them from an application configuration file.
+
+```
+// instantiate the module configuration
+var metricsCapturingModuleConfiguration = new MetricsCapturingModuleConfigurationParameters(true, true, true, true);
```

Comment:
Aren't there only 3 boolean flags in this constructor, instead of 4?

Reply:
Yep!  Fixed.